test/test_fx_passes.py

```diff
@@ -664,6 +664,25 @@ def pattern(a, a_dtype, a_scale, b, b_dtype, b_scale, out_scale):
         TestCase(False, False, 1),
     ]
 
+class NoAnchorFound:
```

Comment:
can we add some comment talking about what no anchor found means?

Reply:
Updated.